demo/docker-compose.alice-bob.yml

```diff
@@ -33,3 +33,25 @@ services:
     ports:
       - "3003:3003"
     command: poetry run aca-py start --arg-file ./configs/bob.yml 
+
+  ngrok-tails-server:
+    image: reflectivedevelopment/agent-tunnel
+#    networks:
+#      - tails-server
+    ports:
+      - 4044:4040
+    command: tails-server:6543 -p 4040 --host ${AGENT_TUNNEL_HOST}
+  tails-server:
+    image: docker_tails-server
```

Comment:
Given that there isn't a published version of this anywhere, I went ahead and published one to docker hub as `dbluhm/indy-tails-server:latest`. Not an ideal location but this way we can make it so the end demo user doesn't have to build the image themselves. I think I recommend using this published image for now.

Reply:
Great catch! I hadn't noticed this while working day-in/day-out on my machine. Ah, the wonders of docker, where you can't just "run from scratch" every time and ensure that all the containers you have are fresh xP

Reply:
Thank you for noticing this, I have changed the compose file to point to `dbluhm/indy-tails-server:latest` in the latest commits.